index.html

```diff
@@ -1,10 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/navbar.css">
     <meta charset="UTF-8">
-    <title>Better than Apple</title>
+    <title>let fake = (🍎) =>🍋</title>
 </head>
 <body>
-        
+    <main id="hero" class="container">
```

Comment:
[Optional] I suggest wrapping all your content in the main tag

Reply:
done